src/pixel/state.js

```diff
@@ -154,11 +157,20 @@ export function StateWrapper (state) {
    * @private
    */
   function _asTuples () {
+    return _tuples(_pMap).concat(_tuples(_pMapLowPriority))
+  }
+
+  /**
+   * @param {Map} parameterMap
+   * @returns {string [][]}
+   * @private
+   */
+  function _tuples (parameterMap) {
     let array = []
     Object.keys(_state).forEach((key) => {
       const value = _state[key]
-      if (_pMap[key]) {
-        const params = _pMap[key](value)
+      if (parameterMap[key]) {
```

Comment:
I would just make `_pMap` a list of tuples instead of a map. Then the order is preserved. And implement this method by traversing _pMap instead of the state.

Reply:
This should work. Let me change that.